.github/workflows/bat.yml

```diff
@@ -63,6 +63,11 @@ jobs:
             test-results-simulink-test: test-results/simulinktest.mldatx
             model-coverage-cobertura: test-results/modelcoverage.xml
             test-results-pdf: test-results/pdf-results.pdf
+            test-results-junit: test-results/results.xml
+            code-coverage-cobertura: test-results/coverage.xml
+            code-coverage-html: test-results/codecoverageHTML
+            model-coverage-html: test-results/modelcoverageHTML
+            test-results-html: test-results/resultsHTML
```

Comment:
Providing a folder is fine but I'm thinking should we use `test-results/codecoverage.html`, `test-results/modelcoverage.html` and `test-results/html-results.html` respectively for consistency